{
    "Adds_Arcane Sigil_Attack_Esoteric Dyad": "エソテリックダイアド",
    "Adds_Arcane Sigil_Attack_Esoteric Ray": "エソテリックレイ",
    "Adds_Arcane Sigil_Attack_Esoteric Sect": "エソテリックセクト",
    "Adds_Arcane Sigil_Name": "秘紋",
    "Adds_Arcane Sigil_Status_Vulnerability Up_Desc": "受けるダメージが増加する状態。",
    "Adds_Arcane Sigil_Status_Vulnerability Up_Name": "被ダメージ上昇",
    "Adds_Behemoth_Attack_Adikia": "アディキア",
    "Adds_Behemoth_Attack_Meteoros Eidolon": "メテオロス・エイドロン",
    "Adds_Behemoth_Attack_Styx": "ステュクス",
    "Adds_Behemoth_Attack_Unknown_664F": "Unknown_664F",
    "Adds_Behemoth_Name": "ベヒーモス",
    "Adds_Behemoth_Status_Vulnerability Up_Desc": "受けるダメージが増加する状態。",
    "Adds_Behemoth_Status_Vulnerability Up_Name": "被ダメージ上昇",
    "Adds_Python_Attack_Explosion": "爆散",
    "Adds_Python_Attack_Opheos Eidolon": "オフェオス・エイドロン",
    "Adds_Python_Attack_Phlegethon": "プレゲトン",
    "Adds_Python_Attack_Styx": "ステュクス",
    "Adds_Python_Attack_Unknown_6650": "Unknown_6650",
    "Adds_Python_Name": "ピュトン",
    "Adds_Python_Status_Vulnerability Up_Desc": "受けるダメージが増加する状態。",
    "Adds_Python_Status_Vulnerability Up_Name": "被ダメージ上昇",
    "Bosse_Unknown Source_Name": "Unknown Source",
    "Bosse_Unknown Source_Status_Fetters_Desc": "行動できなくなった状態。",
    "Bosse_Unknown Source_Status_Fetters_Name": "拘束",
    "Bosse_Unknown Source_Status_Tenebrous Grasp_Desc": "移動できなくなった状態。",
    "Bosse_Unknown Source_Status_Tenebrous Grasp_Name": "闇の呪縛",
    "Bosse_Zodiark_Attack_Adikia": "アディキア",
    "Bosse_Zodiark_Attack_Algedon": "アルゲドン",
    "Bosse_Zodiark_Attack_Ania": "アニア",
    "Bosse_Zodiark_Attack_Astral Eclipse": "アストラルエクリプス",
    "Bosse_Zodiark_Attack_Astral Flow": "アストラルフロウ",
    "Bosse_Zodiark_Attack_Attack": "攻撃",
    "Bosse_Zodiark_Attack_Complete Control": "完全接続",
    "Bosse_Zodiark_Attack_Exoterikos": "エクソーテリコス",
    "Bosse_Zodiark_Attack_Explosion": "爆散",
    "Bosse_Zodiark_Attack_Kokytos": "コキュートス",
    "Bosse_Zodiark_Attack_Meteoros Eidolon": "メテオロス・エイドロン",
    "Bosse_Zodiark_Attack_Opheos Eidolon": "オフェオス・エイドロン",
    "Bosse_Zodiark_Attack_Paradeigma": "パラデイグマ",
    "Bosse_Zodiark_Attack_Phlegethon": "プレゲトン",
    "Bosse_Zodiark_Attack_Styx": "ステュクス",
    "Bosse_Zodiark_Attack_Trimorphos Exoterikos": "トライ・エクソーテリコス",
    "Bosse_Zodiark_Attack_Triple Esoteric Ray": "トライ・エソテリックレイ",
    "Bosse_Zodiark_Attack_Unknown_67DD": "Unknown_67DD",
    "Bosse_Zodiark_Name": "ゾディアーク",
    "Bosse_Zodiark_Status_Vulnerability Up_Desc": "受けるダメージが増加する状態。",
    "Bosse_Zodiark_Status_Vulnerability Up_Name": "被ダメージ上昇",
    "ContentDesc_Prophetie - Zodiark": "それは悪意か、絶望か――封印が破られ目覚めの途にあったゾディアークが、ファダニエルの手に落ちた。古の昔、新たな星の理を紡ぎ、世界を救わんと願った人々の想いは踏みにじられ、今はただ滅びを呼ぶ傀儡と化した。すべての鏡像世界を統合し、今を生きる人々の生命を代償として、古の人々を復活させるという彼らの願いは受け入れられない。だが、彼らの想いを覚えていると誓った身なればこそ、ゾディアークに星を壊させてはならない！",
    "ContentName_The Dark Inside": "月の底",
    "QuestLocation_Mare Lamentorum (25.2, 30.8)": "嘆きの海 (25.2, 30.8)",
    "QuestNPC_lustrous dog": "光を帯びた犬",
    "QuestName_The Martyr": "星を救った英雄",
    "instancecontenttextdata_ids_28000_Text": "どうせ終わるなら、派手に散ろうじゃありませんか！",
    "instancecontenttextdata_ids_28001_Text": "ゾディアークと繋がりきるまで、あと少し……！",
    "instancecontenttextdata_ids_28002_Text": "ああ、ついに、このときが……！",
    "instancecontenttextdata_ids_28003_Text": "ハハハハハハハッ！",
    "instancecontenttextdata_ids_28007_Text": "活なる闇で、新たなる理を紡がん……",
    "instancecontenttextdata_ids_28008_Text": "天地万象を我が意のままに！",
    "instancecontenttextdata_ids_28009_Text": "イデア展開……獣よ来たれ……！",
    "instancecontenttextdata_ids_28010_Text": "さて、終末の再現を始めましょう……",
    "instancecontenttextdata_ids_28011_Text": "恐怖よ、星となれ……！",
    "instancecontenttextdata_ids_28012_Text": "降り注げ、あの日のように……！",
    "instancecontenttextdata_ids_28013_Text": "この魔力、シルクスの塔が玩具に思える……！",
    "instancecontenttextdata_ids_28014_Text": "先などないのだ、出し惜しみはしない！",
    "instancecontenttextdata_ids_28018_Text": "耐えられますか！",
    "instancecontenttextdata_ids_28019_Text": "この日を待ち望んでいたのだ！",
    "instancecontenttextdata_ids_28020_Text": "すべてを終わらせるために！",
    "instancecontenttextdata_ids_28021_Text": "完全に繋がった、私こそがゾディアークだ！",
    "tt_card_Zodiark": "ゾディアーク"
}